detox/e2e/test/channels/create_public_channel.e2e.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {toChannelScreen} from '@support/ui/screen';
+
+import {Setup} from '@support/server_api';
+
+describe('Channels', () => {
+    beforeAll(async () => {
+        const {user} = await Setup.apiInit();
+
+        await toChannelScreen(user);
+    });
+
+    it('MM-T3201 Create public channel', async () => {
+        // # Go to channel sidebar list
+        await element(by.id('channel_drawer_button')).tap();
```

Comment:
We recently merged a [PR adding ID to the same button](https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/channel/channel_nav_bar/channel_drawer_button/channel_drawer_button.js#L137). With that, please update to
```suggestion
        await element(by.id('channel_drawer.button')).tap();
```

Reply:
done!